adoptedStyleSheets.js

```diff
@@ -100,6 +118,28 @@
 
     CSSStyleSheet.prototype.replaceSync = replaceSync;
     StyleSheet.prototype.replaceSync = replaceSync;
+    
+    
+    function hookCSSStyleSheetMethod(/** @type {keyof typeof CSSStyleSheet.prototype} */ key) {
+        const old = CSSStyleSheet.prototype[key]
+        CSSStyleSheet.prototype[key] = function hook(...args) {
+            /** @type {_StyleSheet | CSSStyleSheet} */
+                if (node in this) {
+                  this[node]._adopters.forEach(i => {
```

Comment:
Can we expand out this variable name (`i`) so it's clear what we're talking about? Maybe `adopter`? Minifying will take care of reducing that variable eventually.